x-pack/test/examples/search_examples/search_example.ts

```diff
@@ -16,12 +16,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const comboBox = getService('comboBox');
   const toasts = getService('toasts');
 
-  describe('Search session example', () => {
+  describe('Search example', () => {
     const appId = 'searchExamples';
 
     before(async function () {
       await PageObjects.common.navigateToApp(appId, { insertTimestamp: false });
-      await comboBox.set('indexPatternSelector', 'logstash-*');
```

Comment:
I found that `set` didn't play nice with a single selection combo box with a default non-clearable value. It broke in case `logstash-*` was already selected